manifests/argo-rollouts.yaml

```diff
@@ -0,0 +1,291 @@
+apiVersion: v1
```

Comment:
Can we automatically insert warning headers that are generated by make ?

Reply:
Yeah, I think we can achieve that at `make` level (use `sed` I guess). Looks like `kustomize` doesn't support adding a header to the build result.

Reply:
This is done